### ISSUE TITLE
Multiline selection option is reset by Tools -> Options

### ISSUE THREAD

**Issue by int19h** [Microsoft/RTVS#735]
Start with the multiline switch enabled in History:
![image](https://cloud.githubusercontent.com/assets/1175142/12634471/a38ab280-c52e-11e5-85e9-2dafce5ec896.png)

Options looks right:
![image](https://cloud.githubusercontent.com/assets/1175142/12634477/b16a9c6c-c52e-11e5-901a-ce3368c1fe94.png)

Now go back to History and toggle it:
![image](https://cloud.githubusercontent.com/assets/1175142/12634482/c0319944-c52e-11e5-8686-d9fce613a47d.png)

Options reflects the change:
![image](https://cloud.githubusercontent.com/assets/1175142/12634493/d13f9cfe-c52e-11e5-992e-504f6194c39d.png)

Now press "Cancel" to exit Options. The switch resets back to `True`!
![image](https://cloud.githubusercontent.com/assets/1175142/12634495/df5b9a04-c52e-11e5-8a49-e9680c50060c.png)

Re-opening Options again confirms it:
![image](https://cloud.githubusercontent.com/assets/1175142/12634501/e97a8d9c-c52e-11e5-8b3e-7957a31d93fe.png)

Basically, when this option is changed via the toggle on the toolbar, opening Options and then closing it with Cancel will reset the switch to the last value that was saved via Options.
